packages/e2e-test-app/windows/ReactUWPTestApp/ReactUWPTestApp.csproj

```diff
@@ -136,6 +136,9 @@
     <PackageReference Include="XamlTreeDump">
       <Version>1.0.9</Version>
     </PackageReference>
+    <PackageReference Include="ReactNative.Hermes.Windows">
+      <Version>0.8.0-ms.0</Version>
+    </PackageReference>
```

Comment:
curious why we need this here? shouldn't this come in as part of the central property sheets if the app chooses UseHermes?
/ is adding the reference enough to mean "use hermes", or do we still need the ExperimentalFeatures.props bit?

Reply:
ExperimentalFeatures.props bit is still required (or setting `UseHermes property` somewhere else) to "use" hermes .. This PR ensures that the hermes integration is available in the Microsoft.ReactNative binaries ( contolled by new `includeHermes` property ) so that the dynamic switching works,.

This should go to a common place, but I can't find the right place .. The same is true for the references above such as `Microsoft.NETCore.UniversalWindowsPlatform` .. Please suggest one if it's available .. Otherwise i think we need to do some refactoring ..

Reply:
These are normally added in the new project template.

Reply:
For the tests projects checked into the repo, is there a place with common definitions ?